<launch>

  <!-- Shortcuts -->
  <arg name="model" default="$(find hw1)/urdf/model.urdf"/>
  <arg name="rvizconfig" default="$(find hw1)/rviz/urdf.rviz" />

  <!-- Some magic -->
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />

  <!-- Run nodes -->
  <node name="joint_state_publisher" pkg="hw1" type="script.py" output="screen" />

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />


</launch>

